### DISCUSSION TITLE
Can i add HTML to email notifications templates?

### DISCUSSION THREAD

**Discussion by chateaufiesta** [globaleaks/globaleaks-whistleblowing-software#4555] [Q&A]
**Is your feature request related to a problem? Please describe.**
We would like to be able to "beautify" the notifications templates by using html (img, etc)

**Describe the solution you'd like**
Add an option to the notification settings where we can choose if the notifications are sent in plain text mode or html mode.

**Describe alternatives you've considered**
The alternative is to have a external job running in the background and send the notifications from there.

**Additional context**
Example of a notification (tip mail template) that doesn't work on plain text (tags img, em and a href not working)
""
&lt;img src="https://xxxx/s/logo"/&gt;

This is an email to notify you that a whistleblower has selected you as a valuable recipient of their &lt;em&gt;report&lt;/em&gt;.

You are asked to pay special attention to the information and material contained therein. Please keep in mind that whistleblowers often expose themselves to high personal risks in the interest of the public. Therefore the material provided within this report should be considered to be of high importance.

submission: https://xxx/#/status/31871335-5c68-4bb1-ad45-89b6510620bd

bla bla bla &lt;a href="https://xxx/s/declaracao.pdf"&gt;herei&lt;/a&gt;.
""

**Comment by evilaliv3** [comment]
Closing as duplicated for https://github.com/globaleaks/GlobaLeaks/issues/370

**Comment by evilaliv3** [comment]
Thank you for your feedback @chateaufiesta.

Actually at thew moment emails are sent in plain text intentionally in order to ensure security.

We have an ongoing penetration test that have just verified this aspect and they were happy that the application do not support for HTML.

@groke: what do you think about this possible extention ?
